kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -257,12 +310,18 @@
   /**
    * Update a {@link Dataset} for the given dataset or view URI.
    * <p>
-   * URI formats are defined by {@code Dataset} implementations, but must begin
-   * with "dataset:" or "view:".
+   * You can add, remove, or change the datatype of columns in your dataset,
```

Comment:
Reworded to 

   * You can add columns, remove columns, or change the data type of columns 
   * in your dataset, provided you don't attempt a change that would corrupt 
   * the data.